propka/molecular_container.py

```diff
@@ -117,6 +118,16 @@ def average_of_conformations(self):
             self.conformation_names[0]].chains
         self.conformations['AVR'] = avr_conformation
 
+    def write_propka(self, filename=None):
+        """Write PROPKA input file.
+
+        Args:
+            filename:  file to write to
+        """
+        if filename is None:
+            filename = os.path.join('{0:s}.propka'.format(self.name))
```

Comment:
So the extension is now `.propka`, and not `.propka_input` anymore? I'm not familiar with these files, I just noticed the change.

Reply:
Thank you; I missed this as well.  Will fix.